Backend/app/models/customer.model.js

```diff
@@ -2,10 +2,24 @@ const sql = require("./db.js");
 
 // constructor
 const Customer = function(customer) {
-  this.email = customer.email;
+  this.user_id = customer.user_id;
   this.name = customer.name;
-  this.active = customer.active;
+  this.email = customer.email;
+  this.phone = customer.phone;
 };
 
 
+Customer.retrieveAll = function() {
+  return new Promise(function(resolve, reject){
+    sql.query('SELECT * FROM client_users', function(err, rows) {
+      if (err) throw err;
```

Comment:
Use reject() for the error instead of throwing.

Reply:
Good point, thanks!